GW2EIEvtcParser/EncounterLogic/Strikes/SotO/TempleOfFebe.cs

```diff
@@ -208,6 +209,16 @@ internal override List<PhaseData> GetPhases(ParsedEvtcLog log, bool requirePhase
                 var phase = new PhaseData(enragedSmash.Time, log.FightData.FightEnd, "Enraged Smash");
                 phase.AddTarget(mainTarget);
                 phases.Add(phase);
+                // Sub Phase for 50-10%
+                for (int i = 1; i <= invulnPhases.Count; i++)
```

Comment:
var phase3 = invulPhases.LastOrDefault(x => x.InInterval(enragedSmash.Time));
null check on phase3 then use its start.
So no failed breakbar phases?

Reply:
No just 50-10 until the breakbar finally breaks